### ISSUE TITLE
a11y: group image should be descriptive for screen reader users

### ISSUE THREAD

**Issue by Danielle254** [LetsGetTechnical/elecretanta#534]
![Image](https://github.com/user-attachments/assets/5469ac06-b591-4adc-9264-4b04e597cb43)

Current state: image alt text is created from combining the group title with the word image. For example: "Trip to Florida image". This is not descriptive of what is actually contained within the image and excludes screen reader users from experiencing the website.

Fix: create appropriate alt text to describe the image. This should be the same alt text in all places the images are used (create-group page, edit group, etc).
